tests/tests.py

```diff
@@ -257,6 +257,22 @@ def test_count_with_filter(self):
         vc.append_column(SumColumn('indicator_a'))
         self.assertEqual(1, vc.count(self.session.connection(), {'username': 'user1'},))
 
+    def test_count_no_grouping(self):
+        vc = QueryContext(
+            "user_table",
+            filters=[EQ('user', 'username')],
+        )
+
+        vc.append_column(SimpleColumn('indicator_a'))
+        self.assertEqual(2, vc.count(self.session.connection(), {'username': 'user1'},))
+
+    def test_count_with_nulls(self):
+        vc = QueryContext(
+            "user_table",
+        )
+
+        self.assertEqual(4, vc.count(self.session.connection(),))
```

Comment:
`user_table` doesn't have any NULLed columns, do they? That's why I didn't write a test.

Reply:
yes it does - `indicator_{c|d}` are only defined on 2 out of the 4 rows. I ran the test with the old code and it returned `2`

Reply:
Cool, makes sense. I thought the query picked username column which is not NULL